integration-tests/smoke/keeper_test.go

```diff
@@ -115,549 +115,538 @@ func getKeeperSuite(
+					"replicas": "5",
+					"env": map[string]interface{}{
+						"MIN_INCOMING_CONFIRMATIONS": "1",
+						"KEEPER_TURN_FLAG_ENABLED":   "true",
+						"KEEPER_TURN_LOOK_BACK":      "0",
+					},
+				}))
+			err = testEnvironment.Run()
+			Expect(err).ShouldNot(HaveOccurred())
+
+			By("Connecting to launched resources")
+			chainClient, err = blockchain.NewEthereumMultiNodeClientSetup(networks.SimulatedEVM)(testEnvironment)
+			Expect(err).ShouldNot(HaveOccurred(), "Connecting to blockchain nodes shouldn't fail")
+			contractDeployer, err = contracts.NewContractDeployer(chainClient)
+			Expect(err).ShouldNot(HaveOccurred(), "Deploying contracts shouldn't fail")
+			chainlinkNodes, err = client.ConnectChainlinkNodes(testEnvironment)
+			Expect(err).ShouldNot(HaveOccurred(), "Connecting to chainlink nodes shouldn't fail")
+			chainClient.ParallelTransactions(true)
+
+			By("Funding Chainlink nodes")
+			txCost, err := chainClient.EstimateCostForChainlinkOperations(1000)
+			Expect(err).ShouldNot(HaveOccurred(), "Estimating cost for Chainlink Operations shouldn't fail")
+			err = actions.FundChainlinkNodes(chainlinkNodes, chainClient, txCost)
+			Expect(err).ShouldNot(HaveOccurred(), "Funding Chainlink nodes shouldn't fail")
+
+			By("Deploy Keeper Contracts")
+			linkToken, err = contractDeployer.DeployLinkTokenContract()
+			Expect(err).ShouldNot(HaveOccurred(), "Deploying Link Token Contract shouldn't fail")
+
+			switch consumerContract {
+			case BasicCounter:
+				registry, registrar, consumers, upkeepIDs = actions.DeployKeeperContracts(
+					registryVersion,
+					registryConfig,
+					defaultUpkeepsToDeploy,
+					defaultUpkeepGasLimit,
+					linkToken,
+					contractDeployer,
+					chainClient,
+					linkFundsForEachUpkeep,
+				)
+			case PerformanceCounter:
+				registry, registrar, consumersPerformance, upkeepIDs = actions.DeployPerformanceKeeperContracts(
+					registryVersion,
+					defaultUpkeepsToDeploy,
+					defaultUpkeepGasLimit,
+					linkToken,
+					contractDeployer,
+					chainClient,
+					&registryConfig,
+					linkFundsForEachUpkeep,
+					10000,   // How many blocks this upkeep will be eligible from first upkeep block
+					5,       // Interval of blocks that upkeeps are expected to be performed
+					100000,  // How much gas should be burned on checkUpkeep() calls
+					4000000, // How much gas should be burned on performUpkeep() calls. Initially set higher than defaultUpkeepGasLimit
+				)
+			}
 
-			By("Register Keeper Jobs", func() {
-				actions.CreateKeeperJobs(chainlinkNodes, registry)
-				err = chainClient.WaitForEvents()
-				Expect(err).ShouldNot(HaveOccurred(), "Error creating keeper jobs")
-			})
+			By("Register Keeper Jobs")
+			actions.CreateKeeperJobs(chainlinkNodes, registry)
+			err = chainClient.WaitForEvents()
+			Expect(err).ShouldNot(HaveOccurred(), "Error creating keeper jobs")
 		})
 
-		Describe("with Keeper job", func() {
+		DescribeTable("with Keeper job", func() {
```

Comment:
I'm not sure if DescribeTable has any difference than Describe in its current form. Looking at [example](https://pkg.go.dev/github.com/onsi/ginkgo/extensions/table) it seems to be useful when the function within Describe has some parameters, and you can specify these parameters as different table entries